theme/less/components/read-more.less

```diff
@@ -0,0 +1,33 @@
+.read-more {
+  position: relative;
+  max-height: 284px;
+  overflow: hidden;
+  transition:max-height 0.15s ease-out;
+}
+
+.read-more.expand {
+  max-height: 5000px !important;
+  transition:max-height 0.35s ease-in-out;
+}
```

Comment:
Just FYI, transitioning on unknown max-height has a lot of problems : delays, incorrect timing and potential ugly transitions (see : https://stackoverflow.com/a/8331169). I created a small vanilla utility which uses Popmotion for this : https://github.com/etalab/udata-front/blob/master/theme/js/components/vanilla/accordion.js

Reply:
I'll look at it !